src/smartpeak/source/core/RawDataProcessor.cpp

```diff
@@ -75,7 +75,23 @@ namespace SmartPeak
 
   ParameterSet RawDataProcessor::getParameterSchema() const
   {
-    return ParameterSet();
+    std::map<std::string, std::vector<std::map<std::string, std::string>>> param_struct({
+    {"mzML", {
+      {
+        {"name", "format"},
+        {"type", "string"},
+        {"value", "Chromeleon"},
+        {"description", ""},
```

Comment:
We can add the following to the description:

"XML files are those with the ending `.mzML` or `.mzXML`.  Chromeleon files are text files generating from the Thermo family of HPLCs using the Chromeleon software."